documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -20,36 +27,34 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
   For Linux users: ::
     
     cd RMG-Py
+    source ~/.bashrc
```

Comment:
Most of the time, users install Anaconda and do not open a new terminal before trying to create the conda environment. This causes an error because while anaconda was added to their path, they need to re-source their .bashrc file for bash to recognize the `conda` command. There is never any harm in re-sourcing your .bashrc file, and this saves new bash users some headache. Experienced Linux users will be able to decide if they need this line or not.

Alternatively, I can just add a note below this saying something like "If you get an error about the command `conda` not being found, then make sure that it is in your .bashrc and resource your .bashrc"

Reply:
OK (no need to add further explanation)